app.yaml

```diff
@@ -30,6 +30,7 @@ handlers:
     Vary: Accept-Encoding
 - url: /build
   static_dir: build
+  application_readable: true
```

Comment:
Yep, that's what we discussed during the meeting. We should avoid referencing directly to build/ as an intermediary step toward a final solution and go straight for the final solution in this PR.